client/src/components/LoginContainer.js

```diff
@@ -0,0 +1,25 @@
+import styled from "@emotion/styled";
+import React from "react";
+import LoginForm from "./LoginForm";
+
+function LoginContainer(props) {
+  return (
+    <Container>
+      <h2>Mega App 2020</h2>
+      <h3>Login and start saving</h3>
+      <LoginForm />
+    </Container>
+  );
+}
+
+export default LoginContainer;
+const Container = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+  height: 70vh;
```

Comment:
It's not a good practice to set `vh` in components. This will make it difficult to adjust single components and create a responsive layout.
Imagine, that you make the header smaller. This would lead to changes in this component too.

In addition, this leads to UI issues like this:
![image](https://user-images.githubusercontent.com/10058950/93422438-15194200-f8b4-11ea-948b-e05c7acbe7c7.png)

Instead of `vh`, you shouldn't limit the height here (or set it to 100%). 
Try to create components independent of the app they are added to. 
This login screen should work in an app without a header element too, thus, there shouldn't be app related CSS inside. This is something the app takes care of.

Same issue in the RegisterContainer.

Reply:
I know. This was just an in between solution as I will work on the specific component today and will take care of the heights. 👍